packages/plugin-core/src/constants.ts

```diff
@@ -281,14 +281,14 @@ export const DENDRON_COMMANDS: { [key: string]: CommandEntry } = {
   },
   LOOKUP_NOTE: {
     key: "dendron.lookupNote",
-    title: `${CMD_PREFIX} Lookup Note`,
+    title: `${CMD_PREFIX} Note Lookup`,
     group: "navigation",
     desc: "Initiate note lookup",
     when: DendronContext.PLUGIN_ACTIVE,
   },
   LOOKUP_SCHEMA_V3: {
```

Comment:
don't forget to delete and to add the keyboard shortcuts back to the new lookup schema

```
  LOOKUP_SCHEMA: {
    key: "dendron.lookupSchema",
    title: `${CMD_PREFIX} Lookup Schema`,
    group: "navigation",
    keybindings: {
      mac: "cmd+shift+L",
      key: "ctrl+shift+l",
      when: DendronContext.PLUGIN_ACTIVE,
    },
    desc: "Initiate schema lookup",
    docLink: "dendron.topic.lookup.md",
    docPreview:
      "![](https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/images/schema-lookup.gif)",
  },
```

Reply:
are we already deprecating v2?

Reply:
reading the comments backwards 😅  just saw the other comment.